### ISSUE TITLE
radius not called in bubble chart

### ISSUE THREAD

**Issue by BenjaminWaye** [jerairrest/react-chartjs-2#562]
Adding `options={{
            elements: {
              point: {
                radius: () => {
                  console.log("blaa");
                },
              },
            },`

blaa doesn't get called. In other words this prevents customisation of the the individual points in the bubble chart.

**Comment by jhonnold** [comment]
According to the [Chart.js documentation](https://www.chartjs.org/docs/latest/configuration/elements.html#point-configuration) this value should be a number and not a function, so what you are looking to do may not be possible with vanilla chart.js. There may be a plugin to accomplish this.

**Comment by BenjaminWaye** [comment]
Hmm interesting. But hard coding radius to e.g. 2 has no effect on the size of the points. Either way I will most likely need a plugin. Still why is this option there if it doesn't work?

**Comment by jhonnold** [comment]
Looking deeper into the documentation I found that it is possible to adjust the radius per bubble. 

See [here](https://www.chartjs.org/docs/latest/charts/bubble.html#data-structure).

Rather than setting a global radius, you either want to set it on each data set or on individual data points.

**Comment by BenjaminWaye** [comment]
Yes you can adjust the radius per bouble but then you need to alter the value of the data labels so that the amount is still correct. I guess it works but messy
